### PR TITLE
Feat/add pr state to contribution fetch job

### PR DESCRIPTION
**Pull request by andreLumor**
Resolves: #238 

What was done:
Added pr_state to contributions model
Updated github contribution fetch to also collect pr_state

To run tests:
`rspec spec/services/github/contributions/wrappers/pull_request_spec.rb:39`

### CODE REVIEW

File: spec/services/github/contributions/wrappers/pull_request_spec.rb [hunk 1]
```diff
@@ -30,4 +35,15 @@
   describe '#created_at' do
     it { expect(subject.created_at).to eq '2022-01-01' }
   end
+
+  describe '#pr_state' do
```
Comment (geeksilva97) [inline spec/services/github/contributions/wrappers/pull_request_spec.rb:39]: Could we have a context for open PRs as well? In order to cover all the possibilities.

File: db/schema.rb [hunk 2]
```diff
@@ -46,6 +46,23 @@
...
+    t.index ["company_id"], name: "index_clients_on_company_id"
+  end
+
+  create_table "companies", id: :serial, force: :cascade do |t|
+    t.string "name"
+    t.datetime "created_at", precision: nil
+    t.datetime "updated_at", precision: nil
+    t.string "avatar"
+    t.integer "end_period"
+  end
```
Comment (geeksilva97) [inline db/schema.rb:64]: Is is related to `pr` state? Wondering if it came from another issue you're working on.

File: spec/services/github/contributions/wrappers/pull_request_spec.rb (hunk 1 above)
Comment (andreLumor) [reply spec/services/github/contributions/wrappers/pull_request_spec.rb:39]: Ok!

File: db/schema.rb (hunk 2 above)
Comment (andreLumor) [reply db/schema.rb:64]: Oh, this wasn't supposed to be there.

**Review by geeksilva97 — APPROVED** [review]

**Pull request merged**